packages/riviere-extract-ts/src/extraction-rules/evaluate-extraction-rule-method.ts

```
@@ -0,0 +1,88 @@
+    value: {
+      parameters,
+      returnType: returnTypeNode?.getText() ?? 'unknown',
+    },
+  }
+}
+
+export function evaluateFromConstructorParamsRule(
+  _rule: FromConstructorParamsExtractionRule,
+  classDecl: ClassDeclaration,
+): MethodExtractionResult {
+  const constructor = classDecl.getConstructors()[0]
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)